microraiden/microraiden/VERSION

```diff
@@ -0,0 +1 @@
+0.2.1
```

Comment:
Maybe we should use a dev version to avoid confusion on bug reports etc. Something like 0.2.1-DEV. Only use proper versions in release commits. Thoughts?

Reply:
for dev version it's usual to append current HEAD checksum (like `0.2.1-a1c8defa)

I gave it a little thought and it may work like this:
1) `setup.py` needs to obtain the version. For packaging, it can check if `VERSION` file exists. In developement, the version will be output of `git describe --tag`.
2) when building the package (`py setup.py sdist`), the script can compile that `VERSION` file. It won't be git-tracked -- it'll be package specific.
3) nice thing about `git describe` is that it returns something like this:
`v0.1.4-15-g7c5925f`
where `v0.1.4` is obviously a tag
`15` is number of commits since the tag
`g7c5925f` is a current commit hash

So for 'proper' pypi releases the reported version will always be simple tag (like `v0.2.1`) and for people using git repo we'll see which commit they are on.